eng/CodeGeneration.targets

```diff
@@ -18,6 +18,12 @@
     <_GenerateCode Condition="'$(_SupportsCodeGeneration)' == 'true' AND '$(AutoRestInput)' != ''">true</_GenerateCode>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TemporaryUsePreviousGeneratorVersion)' == 'true'">
+    <_AutoRestVersion>https://github.com/Azure/autorest/releases/download/autorest-3.0.6223/autorest-3.0.6223.tgz</_AutoRestVersion>
+    <_AutoRestCoreVersion>3.0.6283</_AutoRestCoreVersion>
```

Comment:
I'd suggest pulling out the versions into properties and default them here but allow the individual projects to override them if needed. I suspect overtime we might need to flexibility to pin a given library to a particular generator version.

Reply:
>I suspect overtime we might need to flexibility to pin a given library to a particular generator version

I'd like to postpone this time as much as possible :)